plugins/woocommerce/src/Blocks/BlockTypes/ProductTemplate.php

```diff
@@ -91,15 +91,21 @@ protected function render( $attributes, $content, $block ) {
 			// This ensures that for the inner instances of the Post Template block, we do not render any block supports.
 			$block_instance['blockName'] = 'core/null';
 
+			// Relay the block context to the inner blocks.
+			$block_context = array_merge(
```

Comment:
Nitpick: I think it would be better to rename `$block_context` to `$available_context` as 2nd argument of `WP_Block` constructor is called `available_context`. What do you think?
```suggestion
			$available_context = array_merge(
```